{
  "uid" : "fff46a235d31826b",
  "name" : "test_mul[错误样例]",
  "fullName" : "pytest_practice.testing.test_calculator_second.TestCalculator#test_mul",
  "historyId" : "4eac8e75f8a950e2694cf20077593257",
  "time" : {
    "start" : 1599554105413,
    "stop" : 1599554105414,
    "duration" : 1
  },
  "status" : "failed",
  "statusMessage" : "AssertionError: assert 4 == -3",
  "statusTrace" : "self = <pytest_practice.testing.test_calculator_second.TestCalculator object at 0x111909110>, get_cal = <pytest_practice.pythoncode.calculator.Calculator object at 0x1118a0710>, start_cal = None, get_mul = [3, -1, 4]\n\n        @pytest.mark.mul\n    #    @pytest.mark.run(order=2)\n        @pytest.mark.third\n        def test_mul(self,get_cal,start_cal,get_mul):\n            result = get_cal.mul(get_mul[0],get_mul[1])\n>           assert get_mul[2] == result\nE           assert 4 == -3\n\ntest_calculator_second.py:38: AssertionError",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "start_cal",
    "time" : {
      "start" : 1599554105412,
      "stop" : 1599554105412,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "get_cal",
    "time" : {
      "start" : 1599554105230,
      "stop" : 1599554105230,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "get_mul",
    "time" : {
      "start" : 1599554105413,
      "stop" : 1599554105413,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "testStage" : {
    "status" : "failed",
    "statusMessage" : "AssertionError: assert 4 == -3",
    "statusTrace" : "self = <pytest_practice.testing.test_calculator_second.TestCalculator object at 0x111909110>, get_cal = <pytest_practice.pythoncode.calculator.Calculator object at 0x1118a0710>, start_cal = None, get_mul = [3, -1, 4]\n\n        @pytest.mark.mul\n    #    @pytest.mark.run(order=2)\n        @pytest.mark.third\n        def test_mul(self,get_cal,start_cal,get_mul):\n            result = get_cal.mul(get_mul[0],get_mul[1])\n>           assert get_mul[2] == result\nE           assert 4 == -3\n\ntest_calculator_second.py:38: AssertionError",
    "steps" : [ ],
    "attachments" : [ {
      "uid" : "36e6ff89ab3f5e4f",
      "name" : "stdout",
      "source" : "36e6ff89ab3f5e4f.txt",
      "type" : "text/plain",
      "size" : 64
    } ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 1,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  },
  "afterStages" : [ {
    "name" : "start_cal::0",
    "time" : {
      "start" : 1599554105422,
      "stop" : 1599554105422,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "get_cal::0",
    "time" : {
      "start" : 1599554105468,
      "stop" : 1599554105469,
      "duration" : 1
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "tag",
    "value" : "mul"
  }, {
    "name" : "tag",
    "value" : "run(order=2)"
  }, {
    "name" : "tag",
    "value" : "third"
  }, {
    "name" : "parentSuite",
    "value" : "pytest_practice.testing"
  }, {
    "name" : "suite",
    "value" : "test_calculator_second"
  }, {
    "name" : "subSuite",
    "value" : "TestCalculator"
  }, {
    "name" : "host",
    "value" : "localhost"
  }, {
    "name" : "thread",
    "value" : "17558-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "pytest_practice.testing.test_calculator_second"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ {
    "name" : "get_mul",
    "value" : "[3, -1, 4]"
  } ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ "run(order=2)", "third", "mul" ]
  },
  "source" : "fff46a235d31826b.json",
  "parameterValues" : [ "[3, -1, 4]" ]
}